src/System.Private.CoreLib/src/System/Globalization/ChineseLunisolarCalendar.cs

```diff
@@ -316,19 +316,19 @@ internal override EraInfo[] CalEraInfo
             }
         }
 
-        internal override int GetYearInfo(int LunarYear, int Index)
+        internal override int GetYearInfo(int lunarYear, int index)
         {
-            if ((LunarYear < MIN_LUNISOLAR_YEAR) || (LunarYear > MAX_LUNISOLAR_YEAR))
+            if ((lunarYear < MIN_LUNISOLAR_YEAR) || (lunarYear > MAX_LUNISOLAR_YEAR))
             {
                 throw new ArgumentOutOfRangeException(
-                            "year",
+                            nameof(lunarYear),
```

Comment:
Again, this is helper that is validating argument name for public method. The public method argument name is `year`.